parsec/mca/device/cuda/device_cuda_module.c

```diff
@@ -45,6 +45,8 @@ parsec_cuda_memory_reserve( parsec_device_cuda_module_t* gpu_device,
 static int parsec_cuda_memory_release( parsec_device_cuda_module_t* gpu_device );
 static int parsec_cuda_flush_lru( parsec_device_module_t *device );
 
+extern int parsec_cuda_delegate_task_completion; 
```

Comment:
Can the `extern` definition be moved to a header?

Reply:
I have moved. it to parsec/mca/device/cuda/device_cuda_internal.h